Algorithm/QCAlgorithm.cs

```diff
@@ -267,14 +267,14 @@ public SecurityManager Securities
         }
 
         /// <summary>
-        /// Read-only dictionary containing all active securities. An active security is
```

Comment:
Same comment, "all active securities" does not necessarily need to reference the Security class. Focus on replacing explciit mentions of classes, methods, properties, etc.